check-lint

```diff
@@ -66,15 +66,19 @@ if [ -f "$CONFIG_FILE" ]; then
                 exit 1
             fi
 
-            # Get all Python files in the specified directory
-            FILES=$(git ls-files "$DIRECTORY_PATH/*.py")
```

Comment:
Hi @clebergnu, this solution works, but I would prefer a simpler solution `git ls-files $DIRECTORY_PATH/ | grep .py` what do you think?

Reply:
> Hi @clebergnu, this solution works, but I would prefer a simpler solution `git ls-files $DIRECTORY_PATH/ | grep .py` what do you think?

Works for me as well.